acceptance/analyzer_test.go

```diff
@@ -54,6 +54,10 @@ func TestAnalyzer(t *testing.T) {
 	defer registry.Stop(t)
 
 	os.Setenv("DOCKER_CONFIG", registry.DockerDirectory)
+	// Copy registry docker directory to analyze-image container
+	targetDockerConfig := filepath.Join("testdata", "analyzer", "analyze-image", "container", "registry-docker-config")
```

Comment:
Tests appear to be failing b/c the `registry-docker-config` directory does not exist in the image. Missing `.gitkeep`?

Reply:
Yes, I .gitignore'd it :/